Origin: http://hg.intevation.org/mercurial/crew/rev/7d66a44e87ed
Description: mergetools: refine vimdiff warning message
 We explicitly redraw before echoing the message so that it simply
 displays at the bottom of the window. Also simplifies the message
 printing by using 'echomsg' (which uses 'echohl' internally) and adds
 the names of the software involved for improved Googleability.
Bug: http://bugs.debian.org/701168

diff --git a/contrib/mergetools.hgrc b/contrib/mergetools.hgrc
--- a/contrib/mergetools.hgrc
+++ b/contrib/mergetools.hgrc
@@ -15,7 +15,7 @@ gvimdiff.regkeyalt=Software\Wow6432Node\
 gvimdiff.regname=path
 gvimdiff.priority=-9
 
-vimdiff.args=$local $other $base -c 'echohl WarningMsg | echo "merge conflict detected, type \":cq\" to abort" | echohl'
+vimdiff.args=$local $other $base -c 'redraw | echomsg "hg merge conflict, type \":cq\" to abort vimdiff"'
 vimdiff.check=changed
 vimdiff.priority=-10
 

